doc/api/util.md

```diff
@@ -135,6 +135,13 @@ Each argument is converted to a string using `util.inspect()`.
 util.format(1, 2, 3); // '1 2 3'
 ```
 
+If only one argument is passed to `util.format()`, it is returned as it is
+without any formatting.
+
+```js
+util.format('%% %s') // '%% %s'
```

Comment:
Can you include a semicolon here please? We try to follow the same coding style as the actual codebase.